tests/test_create.py

```diff
@@ -1063,6 +1063,42 @@ def side_effect(self, url, **kwargs):
                 # The mock should have been called with our local channel URL though.
                 assert result_dict.get('local_channel_seen')
 
+    def test_create_from_extracted(self):
+        # Test that we can link a package from its extracted directory
+        # even if the tarball is no longer available.
+        pkgs_dir = PackageCache.first_writable().pkgs_dir
+        mkdir_p(pkgs_dir)
+        pkgs_dir_hold = pkgs_dir + '_hold'
```

Comment:
Yup, exactly.  At some point it would be nice to do an audit of all the packages being used, and figure out if there are instead better packages we could be using just to optimize for time.

One other relevant point... There's a `conda-test` channel we could--and do--use exactly for this.  It's only purpose right now is to hold test packages for conda.